src/js/modules/wallet/controllers/sweep-coins-modal/sweep-coins-modal.ctrl.js

```diff
@@ -0,0 +1,110 @@
+(function () {
+    "use strict";
+
+    angular.module("blocktrail.tools")
+        .controller("SweepCoinsModalController", SweepCoinsModalController);
+
+    function SweepCoinsModalController($scope, $modalInstance, sweeperService, sdkService, Wallet, CONFIG, dialogService,
+                                        $translate, $log) {
+        $scope.bip39EN = blocktrailSDK.bip39wordlist;
+
+        $scope.working = true;
+        $scope.discovering = false;
+        $scope.stepCount = 0;
+
+        $scope.form = {
+            mnemonic: null
+        };
+
+        $scope.sweepData = {
+            rawTx: null,
+            feePaid: null,
+            inputCount: null,
+            totalValue: null
+        };
+
+        var options = {
+            batchSize: 50,
+            accountBatchSize: 1,
```

Comment:
Which should ever only be one account (account index 0) in blockchain.infos case